modules/nw-sriov-supported-devices.adoc

```diff
@@ -5,16 +5,65 @@
 [id="supported-devices_{context}"]
 = Supported devices
 
-{product-title} supports the following network interface controller (NIC) models:
-
-* Intel X710 10GbE SFP+ with vendor ID `0x8086` and device ID `0x1572`
-* Intel XL710 40GbE SFP+ with vendor ID `0x8086` and device ID `0x1583`
-* Intel XXV710 25GbE SFP28 with vendor ID `0x8086` and device ID `0x158b`
-* Mellanox MT27710 Family [ConnectX-4 Lx] 25GbE dual-port SFP28 with vendor ID `0x15b3` and device ID `0x1015`
-* Mellanox MT27800 Family [ConnectX-5] 25GbE dual-port SFP28 with vendor ID `0x15b3` and device ID `0x1017`
-* Mellanox MT27800 Family [ConnectX-5] 100GbE with vendor ID `0x15b3` and device ID `0x1017`
-* Mellanox MT27700 Family [ConnectX-4] VPI adapter card, EDR IB (100Gb/s), single-port QSFP28 with vendor ID `0x15b3` and device ID `0x1013`
-* Mellanox MT27800 Family [ConnectX-5] VPI adapter card, EDR IB (100Gb/s), single-port QSFP28 with vendor ID `0x15b3` and device ID `0x1017`
-* Mellanox MT28880 Family [ConnectX-5 Ex] Ethernet controller, 100Gb/s, dual-port QSPF28 with vendor ID `0x15b3` and device ID `0x1019`
-* Mellanox MT28908 Family [ConnectX-6] VPI adapter card, 100Gb/s (HDR100, EDR IB), single-port QSFP56 with vendor ID `0x15b3` and device ID `0x101b`
-* Mellanox MT28908 Family [ConnectX-6] VPI adapter card, HDR200 IB (200Gb/s), single-port QSFP56 with vendor ID `0x15b3` and device ID `0x101b`
+{product-title} supports the following network interface controllers:
+
+.Supported network interface controllers
+[cols="1,2,1,1"]
+|===
+|Manufacturer |Model |Vendor ID | Device ID 
+
+|Intel
+|X710
+|8088
+|1572
+
+|Intel
+|XL710
+|8088
+|1583
+
+|Intel
+|XXV710
+|8088
+|158b
+
+|Intel
+|E810&#8209;CQDA2
```

Comment:
@zshi-redhat I remembered `E810` is not supported, see https://github.com/openshift/openshift-docs/pull/34860/files

Reply:
@zhaozhanqi  yes, you're right, E810 was removed from the supported list.

@ctauchen  ^^

Reply:
Thanks for that -- I've removed the E810 models. @zhaozhanqi